docs/features.md

```diff
@@ -542,6 +542,53 @@ variations:
         value: '{"title": "Welcome to our website", "subtitle": "We are glad you are here"}'
 ```
 
+## Parents
+
+The concept of parent-child relationship between your features is useful when you wish to control the evaluation of a feature based on the evaluation of another feature.
+
+For example, you may want to expose a feature named `someChild` only if `someParent` is enabled.
```

Comment:
I think it would be nicer to have a real example. Something like `newCheckoutFlow` as the parent and the children could be `newPaymentProvider` and `newShippingFlow`